helios-services/src/main/java/com/spotify/helios/master/MasterService.java

```diff
@@ -201,6 +202,9 @@ public MasterService(final MasterConfig config,
     final ReactorFactory reactorFactory = new ReactorFactory();
     this.rollingUpdateService = new RollingUpdateService(model, reactorFactory);
 
+    // Set up agent reaper (de-registering hosts that have been DOWN for more than X time)
+    this.agentReaper = new DeadAgentReaper(model, config.getAgentReapingTimeout(), TimeUnit.HOURS);
```

Comment:
Nope -- this, that the service doesn't need to be started, is now also covered by the test.